src/main/java/core/basesyntax/reader/CsvFileReader.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax.reader;
+
+import com.opencsv.CSVParser;
+import com.opencsv.CSVParserBuilder;
+import com.opencsv.CSVReader;
+import com.opencsv.CSVReaderBuilder;
+import com.opencsv.exceptions.CsvValidationException;
+import core.basesyntax.exceptions.FileWasNotReadException;
+import core.basesyntax.handler.TransactionHandler;
+import core.basesyntax.handler.impl.BalanceTransactionHandler;
+import core.basesyntax.handler.impl.PurchaseTransactionHandler;
+import core.basesyntax.handler.impl.ReturnTransactionHandler;
+import core.basesyntax.handler.impl.SupplyTransactionHandler;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CsvFileReader {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the beginning of the class definition.